tests/integration/targets/helm/tasks/test_helm_take_ownership.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Test helm take ownership
+  vars:
+    helm_namespace: "{{ test_namespace[13] }}"
```

Comment:
Please add a new namespace here https://github.com/ansible-collections/kubernetes.core/blob/main/tests/integration/targets/helm/defaults/main.yml#L18, otherwise it will fail as no 13th element of the list test_namespace